phost/src/main.rs

```diff
@@ -450,12 +450,7 @@ async fn bridge(args: Args) -> Result<(), Error> {
                 return Err(Error::FailedToCallRegisterWorker);
             }
 
-            let events = ret.unwrap();
```

Comment:
No need these any more because

![image](https://user-images.githubusercontent.com/1024162/87128146-b63ab980-c2c1-11ea-8e6d-230a8dbbb014.png)

now returns `ExtrinsicsSuccess` result, no need to double check events